CondFormats/SiPixelObjects/interface/SiPixelGainForHLTonGPU.h

```diff
@@ -1,6 +1,14 @@
 #ifndef CondFormats_SiPixelObjects_SiPixelGainForHLTonGPU_h
 #define CondFormats_SiPixelObjects_SiPixelGainForHLTonGPU_h
 
+#ifdef __CUDA_ARCH__
+#ifndef GPU_DEBUG
+#ifndef NDEBUG
+#define NDEBUG
+#endif // NDEBUG
+#endif // GPU_DEBUG
+#endif // __CUDA_ARCH__
```

Comment:
Somehow it would feel nice to abstract this piece to a common place, but I'm not sure if, in the end, it would ask more trouble than it would help.